pyaedt/application/Analysis.py

```diff
@@ -479,6 +479,28 @@ def SolutionTypes(self):
         """
         return SOLUTIONS()
 
+    @aedt_exception_handler
+    def get_excitations_name(self):
+        """Get all excitation names.
+
+        Returns
+        -------
+        list
+            List of excitation names. Excitations with multiple modes will return one
+            excitation for each mode.
+
+        References
+        ----------
+
+        >>> oModule.GetExcitations
+        """
+        try:
+            list_names = list(self.oboundary.GetExcitations())
+            del list_names[1::2]
```

Comment:
@maxcapodi78 
Out of curiosity those items of the list?

Reply:
because each excitations return a terminal and and excitation. This is not properly correct. We should consider refactor since on multiple terminals ports that could be wrong